docs/ChangeLog.md

```diff
@@ -1,6 +1,7 @@
 # Splunk Operator for Kubernetes Change Log
 
 ## 0.2.0 Beta (2020-10-15)
+* This release depends upon changes made concurrently in the Splunk Enterprise container images. You must use the latest splunk/splunk:edge nightly image with it, or alternatively any release version 8.1.0 or later.
```

Comment:
Nit: Comment used in other places makes more formal. i.e "*Note that this requires using the Splunk Enterprise container version 8.1.0 or later*"

Reply:
@sgontla I was trying to maintain consistency with Changelog.md entry for 0.1.0 alpha [here](https://github.com/splunk/splunk-operator/blob/develop/docs/ChangeLog.md#010-alpha-2020-03-20).

Reply:
We may need to avoid suggesting the nightly build, if not now, later on, a full release...